data/gegno/ships.txt

```diff
@@ -0,0 +1,954 @@
+	engine 18 152
+	turret 0 -76 "Guile Anti-Missile"
+	turret -44 85.5 "Astuit Battery"
+	turret 44 85.5 "Astuit Battery"
+	leak leak 60 50
+	leak flame 60 80
+	explode "large explosion" 7
+	explode "medium explosion" 20
+	explode "small explosion" 28
+	explode "tiny explosion" 16
+	"final explode" "final explosion medium"
+	description "Felsics are essentially stripped Schists repurposed for a more engaging role. In place of the heavier engines that the Vi had mounted on, they expanded the profile have more efficient engines, dedicating the remaining space to a larger cargo capacity. On top of fitting more cargo room, the Scin also oufitted these 3 turret mounts, allowing them to double as a secondary combat vessel to support their war efforts, and to fill in the lack of a dedicated warship."
+
+ship Feldspar
+	sprite ship/feldspar
+	thumbnail thumbnail/feldspar
+	swizzle 0
+	attributes
+		category "Heavy Warship"
+		licenses
+			"Gegno Civilian"
+			"Scin Architect"
+			"Scin Hoplologist"
+		cost 14760000
+		mass 1146
+		bunks 192
+		"cargo space" 82
+		drag 15
+		"engine capacity" 172
+		"fuel capacity" 600
+		"heat dissipation" 1
+		hull 18800
+		"outfit space" 566
+		"required crew" 133
+		shields 0
+		"weapon capacity" 194
+		weapon
+			"blast radius" 180
+			"shield damage" 1650
+			"hull damage" 1200
+			"hit force" 1000
+	outfits
+		"Acuit Artillery" 2
+		"C27 Campaign Core"
+		Hyperdrive
+		"R03 Warforge Battery"
+		"SC-14 Plasma Steering"
+		"SC-15 Plasma Engines"
+	engine 31 198
+	engine -31 198
+	"reverse engine" -38 -188
+		zoom 0.7
+		angle -360
+	"reverse engine" 38 -188
+		zoom 0.7
+		angle -360
+	"steering engine" 108 33
+		zoom 0.7
+		angle -63.001099
+		left
+	"steering engine" -108 33
+		zoom 0.7
+		angle 63.001099
+		right
+	turret -72.5 17 "Acuit Artillery"
+	turret 72.5 17 "Acuit Artillery"
+	bay Fighter -71 -171.5
+		over
+	bay Fighter 71 -171.5
+		over
+	bay Fighter -69 -97
+		over
+	bay Fighter 69 -97
+		over
+	bay Fighter -65.5 120
+		angle -180
+		over
+	bay Fighter 65.5 120
+		angle -180
+		over
+	leak leak 50 50
+	leak flame 80 140
+	explode "huge explosion" 18
+	explode "large explosion" 30
+	explode "medium explosion" 46
+	explode "small explosion" 50
+	explode "tiny explosion" 40
+	"final explode" "final explosion large"
+	description "The mighty Feldspar is a Scin creation to behold. As a testbed for the new Halite Fighters, Feldspars bring their own small fleets with them and overwhelm their enemy. They also were designed as a semi-experimental platform, built around a brand new multi-engine system and the latest in Gegno plasma technology."
+	description `Able to keep a decent stance away from a battle while putting down artillery fire, Feldspars are quite a dangerously well-rounded ship for their size. On the downside, they are more susceptible to hull damage like any other Scin warship, especially when alone, as they rely a lot on their fighters since they aren't a dedicated warship themselves.`
```

Comment:
```suggestion
	description `	Able to keep a decent stance away from a battle while putting down artillery fire, Feldspars are quite a dangerously well-rounded ship for their size. On the downside, they are more susceptible to hull damage like any other Scin warship, especially when alone, as they rely a lot on their fighters since they aren't a dedicated warship themselves.`
```